chsdi/views/layers.py

```diff
@@ -49,7 +49,7 @@ def layers_config(request):
     params = BaseLayersValidation(request)
     query = params.request.db.query(LayersConfig)
     layers = {}
-    # Python 2/3
+    # TODO: clean-up when only Python 3.x and no longer 2.x is in use
     for layer in get_layers_config_for_params(params, query, LayersConfig):
         # layers = dict(list(layers.items()) + list(layer.items()))
```

Comment:
I think here as well there is nothing to clean up later instead of removing the commented line no 54 that could be done now.